standards/bundle-policy-and-code.md

```diff
@@ -33,4 +39,10 @@ order: 2
 
 ## Developers and designers: what you need to do
 
+* Developers SHOULD become conversant in the process modeling notation that the policy makers in your organization use.  
+* Developers SHOULD track any changes that they make to codebase functionality in the corresponding policy notation for review by policy-makers.
```

Comment:
Perhaps rephrase this to reflect that these two always need to be in sync?

Is the question here that they should get their changes reviewed by policy makers or vice versa?